python/pyspark/sql/tests/test_functions.py

```diff
@@ -310,6 +310,31 @@ def test_input_file_name_udf(self):
         file_name = df.collect()[0].file
         self.assertTrue("python/test_support/hello/hello.txt" in file_name)
 
+    def test_percentile_approx(self):
+        from pyspark.sql.functions import col, percentile_approx
+        from itertools import chain
+        import re
+
+        actual = list(chain.from_iterable([re.findall("(percentile_approx\(.*\))", str(x)) for x in [
+            percentile_approx(col("foo"), 0.5),
+            percentile_approx(col("bar"), 0.25, 42),
+            percentile_approx(col("bar"), [0.25, 0.5, 0.75]),
+            percentile_approx(col("foo"), [0.05, 0.95], 100),
+            percentile_approx("foo", 0.5),
+            percentile_approx("bar", [0.1, 0.9], 10),
+        ]]))
+
+        expected = [
+            "percentile_approx(foo, 0.5, 10000)",
+            "percentile_approx(bar, 0.25, 42)",
+            "percentile_approx(bar, [0.25,0.5,0.75], 10000)",
+            "percentile_approx(foo, [0.05,0.95], 100)",
+            "percentile_approx(foo, 0.5, 10000)",
+            "percentile_approx(bar, [0.1,0.9], 10)"
+        ]
+
+        self.assertListEqual(actual, expected)
```

Comment:
Same as in case of R we want to check if data is properly passed to JVM.

Reply:
One doctest is good enough. I think It's fine to remove this test.